stan/math/rev/scal/fun/falling_factorial.hpp

```diff
@@ -39,35 +35,13 @@ namespace stan {
                - digamma(avi_->val_ - bd_ + 1));
         }
       };
-
-      class falling_factorial_dv_vari : public op_dv_vari {
-      public:
-        falling_factorial_dv_vari(double a, vari* bvi) :
-          op_dv_vari(falling_factorial(a, bvi->val_), a, bvi) {
-        }
-        void chain() {
-          bvi_->adj_ += adj_
-            * val_
-            * digamma(ad_ - bvi_->val_ + 1);
-        }
-      };
     }
 
     inline var falling_factorial(const var& a,
-                                 double b) {
+                                 int b) {
```

Comment:
These can all just go on one line if they fit in 80 characters, which they will here.